cloudinit/ssh_util.py

```diff
@@ -44,7 +44,7 @@
 DISABLE_USER_OPTS = (
     "no-port-forwarding,no-agent-forwarding,"
     "no-X11-forwarding,command=\"echo \'Please login as the user \\\"$USER\\\""
-    " rather than the user \\\"$DISABLE_USER\\\".\';echo;sleep 10\"")
+    " rather than the user \\\"$DISABLE_USER\\\".\';echo;sleep 10;exit 142\"")
```

Comment:
Sorry to keep back and forth.  I'm fine with the solution, but can we do something like this:

    _DISABLE_USER_SSH_EXIT = 142

    # caller replaces $USER and $DISABLE_USER
    DISABLE_USER_OPTS = (
        "no-port-forwarding,no-agent-forwarding,"
        "no-X11-forwarding,command=\"echo \'Please login as the user \\\"$USER\\\""
        " rather than the user \\\"$DISABLE_USER\\\".\';echo;sleep 10;"
        "exit " + str(_DISABLE_USER_SSH_EXIT) + "\"")

Reply:
I don't see why not. I'll send a final commit.